docs/knowledge_base/policy_management.md

```diff
@@ -0,0 +1,13 @@
+# Policy Management
+
+## Add custom policy
+Phylum allows group administrators to upload and apply custom policies. Whether added via the project or group view, these policies are stored at the group level and available to all projects in the group.
+  
+![Project custom policy button](../../assets/project_custom_policy_button.png)
+
+The Phylum UI uses the metadata at the top of the rego to display a `title` and `description` for the policy list, so this data is highly recommended.
```

Comment:
```suggestion
The Phylum UI uses the metadata at the top of the custom policy (rego file) to display a `title` and `description` for the policy list, so this data is highly recommended.
```
This is just a simple suggestion...but more to point out that "rego" and "custom policy" have not exactly been linked and the term may be confusing.

Reply:
both custom and phylum policies are regos... `title` and `description` rule applies to all policies

Reply:
perhaps this?

The Phylum UI uses the metadata at the top of the ~~custom~~ policy file (rego)  to display a `title` and `description` for the policy list, so this data is highly recommended.

Reply:
Roger that...suggestion updated.